pkgdb2client/cli.py

```diff
@@ -214,10 +212,12 @@ def setup_parser():
                         "certificates")
     parser.add_argument('--pkgdburl',
                         help="Base url of the pkgdb instance to query.")
-    parser.add_argument('--fasurl',
+    parser.add_argument('--fasurl', default='https://admin.fedoraproject.org/accounts',
                         help="Base url of the FAS instance to query.")
-    parser.add_argument('--bzurl',
+    parser.add_argument('--bzurl', default='https://bugzilla.redhat.com/xmlrpc.cgi',
```

Comment:
It might be good to make these default URLs a global variable so it doesn't have to be repeated everywhere.